react/features/base/participants/reducer.js

```diff
@@ -59,9 +61,11 @@ const DEFAULT_STATE = {
     fakeParticipants: new Map(),
     haveParticipantWithScreenSharingFeature: false,
     local: undefined,
+    localScreenShare: undefined,
     pinnedParticipant: undefined,
     raisedHandsQueue: [],
     remote: new Map(),
+    sortedFakeScreenShareParticipants: new Map(),
```

Comment:
A better name might be `sortedRemoteFakeScreenShareParticipants`. I didn't want to use the existing `sortedRemoteParticipants` map because I want to be explicit that these are fake participants. This becomes helpful when we need to know the number of participants excluding fakes. 

This is currently a map where the key is the participant id of the owner of the SS and the value is the SS participant. I'm thinking we change this to match the structure of `sortedRemoteParticipants` where the key is the participantId and value is display name

Reply:
If `sortedFakeScreenShareParticipants` does not include the local one then yes call it `sortedRemoteFakeScreenShareParticipants`. Good thinking about decoupling.

>This is currently a map where the key is the participant id of the owner of the SS and the value is the SS participant. I'm thinking we change this to match the structure of sortedRemoteParticipants where the key is the participantId and value is display name

Yes, no need to store the entire participant structure if the data is not used and yes it's good to change to match `sortedRemoteParticipants`.